packages/n/nghttp3/xmake.lua

```diff
@@ -4,7 +4,7 @@ package("nghttp3")
     set_license("MIT")
 
     add_urls("https://github.com/ngtcp2/nghttp3.git")
```

Comment:
We can add tar.gz.

Reply:
We can package sfparse then patch nghttp3 cmake, like this:

https://github.com/xmake-io/xmake-repo/blob/dev/packages%2Fl%2Flsquic%2Fxmake.lua#L26-L34